lib/restforce/collection.rb

```diff
@@ -31,6 +31,11 @@ def size
     end
     alias_method :length, :size
 
+    # Return whether the collection contains any objects.
```

Comment:
I'd prefer to avoid monkey patches, but I'm fine with using `#none?`. There is a performance difference between that and the less flexible `#empty?`, but probably not enough to care about if you'd prefer to maintain a minimal interface. 

Quick benchmark comparing a 1000 calls to `#empty?` vs `#none?` for an empty collection and a collection with about a 1000 sobjects:

```
                        user     system      total        real
empty? with empty:  0.000000   0.000000   0.000000 (  0.000584)
none? with empty:   0.000000   0.000000   0.000000 (  0.001356)
empty? with 1000:   0.000000   0.000000   0.000000 (  0.000399)
none? with 1000:    0.050000   0.000000   0.050000 (  0.052230)
```